src/GameResources.py

```diff
@@ -45,8 +48,15 @@ def update_entity(self, entity: AbstractDungeonEntity) -> None:
                 else:
                     entity.x += x
                     entity.y += y
+
                 entity.ground_symbol = self.level.board[entity.y][entity.x]
                 entity.new_positions = {"x": 0, "y": 0}
+            if str(self.level.board[entity.y + y][entity.x + x]) in ("k"):
```

Comment:
Could we use another symbol instead of k?

Reply:
Yes, we can use any other letter, I just selected K because I was thinking in keys